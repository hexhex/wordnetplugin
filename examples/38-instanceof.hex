%% this returns all words of which a word
%% is instance of

%% the search number for instance of
%% is 38

word("einstein").
instanceof(W,SW,IO) :- &wordnet[38,W](_,SW,IO,_),word(W).
