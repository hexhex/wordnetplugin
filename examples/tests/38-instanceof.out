{instanceof("einstein",1,"physicist"), word("einstein")}
